docs/config.rst

```diff
@@ -380,8 +380,7 @@ Implementation
 
         @Override
         public Raven createRavenInstance(Dsn dsn) {
-            Raven raven = new Raven();
-            raven.setConnection(createConnection(dsn));
```

Comment:
Oops, yeah I meant to delete it. I see no reason for it other than the fact that it didn't exist in the constructor until I added it. This pattern is kind of pervasive in the "old" code -- adding a setter for new fields and not having them in the constructor. I'm not a fan.

But yeah, we didn't deprecate it...